ts/packages/node/src/stores/TomlFile.test.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path';
+import { Config } from '@configu/ts';
```

Comment:
Please make sure to run npm run lint to make sure the code is linted according to the linting rules of the project.